src/seedu/addressbook/Main.java

```diff
@@ -81,6 +83,14 @@ private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
             String userCommandText = ui.getUserCommand();
+            try {
+            	FileReader reader = new FileReader(storage.getPath());
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.